app/src/main/java/example/watarusuzuki/android_example_using_danger/MainActivity.java

```diff
@@ -1,13 +1,37 @@
 package example.watarusuzuki.android_example_using_danger;
 
+import android.app.Activity;
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 
 public class MainActivity extends AppCompatActivity {
 
+    private Intent mIntent;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        while (true) {
+            Intent intent = getIntent();
+            intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+            startActivity(intent);
+            if (intent.hasExtra("hoge")) {
+                convertDaysToMilliseconds(1);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/0c040d149df52a583eb7528530ec4b05dfa97953/app/src/main/java/example/watarusuzuki/android_example_using_danger/MainActivity.java#L22"></span>Return value of convertDaysToMilliseconds(int) ignored, but method has no side effect</td>
    </tr>
  </tbody>
</table>